File: My_Pixels_work/tcp_simulator_with_example/convert_to_DDMMmmmm.py

c = [
    '33.166625, -117.158398',
    '33.167037, -117.157515',
    '33.167482, -117.156817'
    ]


def convert_lat_lng_to_ddmm(lat_lng):
    degrees = int(float(lat_lng))
    hours = float(lat_lng) - degrees
    minutes = abs(hours * 60)

    if minutes < 10:
        minutes = f'0{minutes:.6f}'
    else:
        minutes = f'{minutes:.6f}'

    return f'{int(degrees)}{minutes}'

for item in c:
    lat, lng = item.split(',')
    result = convert_lat_lng_to_ddmm(lat) + ',' + convert_lat_lng_to_ddmm(lng)
    print(result)

File: My_Pixels_work/tcp_simulator_with_example/convert-to-DDMM.mmmm.js
const c = [
    '33.166625, -117.158398',
    '33.167037, -117.157515',
    '33.167482, -117.156817'
]

function convert_lat_lng_to_ddmm(lat_lng) {

    const degrees = parseInt(lat_lng, 10)
    const hours = parseFloat(lat_lng) - degrees
    let minutes = Math.abs(hours * 60);

    if (minutes < 10) {
        minutes = `0${minutes.toFixed(6)}`;
    } else {
        minutes = minutes.toFixed(6);
    }
    return `${degrees.toFixed(0)}${minutes}`;
}

c.map(_ => console.log(convert_lat_lng_to_ddmm(_.split(',')[0]) + ',' + convert_lat_lng_to_ddmm(_.split(',')[1])))
